homu/main.py

```diff
@@ -210,8 +210,12 @@ def sha_or_blank(sha):
     return sha if re.match(r'^[0-9a-f]+$', sha) else ''
 
 def parse_commands(body, username, repo_cfg, state, my_username, db, *, realtime=False, sha=''):
+    try_only = False
     if username not in repo_cfg['reviewers'] and username != my_username:
-        return False
+        if username in repo_cfg['try']:
+            try_only = True
+        else
+            return False
```

Comment:
I'd like to be able to  to throw an "insufficient privileges" error here, too, but I'm not sure how that can be done since I don't fully understand the `state_changed` stuff.

(I'd like to be able to do it without adding a check in every case below)